### DISCUSSION TITLE
Mentioning AKHQ as an alternative to CMAK

### DISCUSSION THREAD

**Discussion by eshepelyuk** [tchiotludo/akhq#632] [Show and tell]
Hi

I'm maintainer of Helm Operator for CMAK, and I've mentioned your project as an alternative on the [CMAK operator homepage](https://github.com/eshepelyuk/cmak-operator#alternatives)

Hope, this is welcomed.

**Comment by tchiotludo** [comment]
Hi @eshepelyuk, 
Clearly a mention / promotion is still welcome ! 
Thanks for that ! 

For my knowledge, what is the main missing feature in AKHQ that exist in CMAK ?

**Reply by eshepelyuk** [reply]
Well, I didnt check the current state but as far as I remember

1. Reassign partitions
1. Display JMX metricsp
1. Display consumers of the topic

**Comment by julienmmm** [comment]
Hello,

I think that the following CMAK feature would be very useful

- increase partition number
- highlight under replicated partition

**Reply by tchiotludo** [reply]
There is this PR for this increase partition number : https://github.com/tchiotludo/akhq/pull/302 never finish ! 
PR are still welcome !
